### DISCUSSION TITLE
在Chrome上養貓！？

### DISCUSSION THREAD

**Discussion by vchien03** [1101-GS4719-C/Python-Community-GS4719-C#1870] [General]
最近Chrome上推出了一個網頁版的「養貓」擴充功能，馬上造成轟動，1週就破2萬下載量，不知道大家有沒有跟上這波養貓潮呢😻
甚至還有不同種類的貓咪、能調整貓咪叫聲、加入低效能的輕量模式等等，愛貓人士們還不跟上！

**Comment by TobywithDino** [comment]
超級可愛...我覺得電子寵物系列都超級療育，明明不像遊戲給人好玩刺激的感覺，卻依然能夠讓人身心舒坦，超級喜歡，我也要來去下載嘞!!!

**Comment by chuanmin0317** [comment]
電子寵物讓我在使用電腦時，感受到了不同的回饋感，有種生活感讓我放鬆心神，不像以往有種冷冰冰的感覺，我蠻有興趣的，馬上去試試看

**Comment by YifenLin** [comment]
這讓我相起曾經風靡一時的popcat：https://popcat.click/
之前高中的時候大家上課都在瘋狂點，讓台灣成為第一名哈哈。
只能說跟貓貓有關的一切都讓人無法抗拒
(我甚至昨天才買了一個貓掌包包XD)

**Comment by Eileen050824** [comment]
對不適合養真實寵物貓的我(會過敏QQ)，這個網頁也算是滿足我一點小心願。網頁版的貓咪省去很多照料的煩惱，雖然可能沒有真實的療癒，但在某方面疲憊的心靈也能得到不少滿足。前提是不要沉迷於此而花太多時間，把它當成舒壓方式就足夠！

**Comment by SusuTima** [comment]
Chrome網頁版的貓咪真的很可愛，這種有互動性的設計感覺也越來越符合當代年輕人的風格。不過其實也可以去貓咖花一點小錢滿足自己想撸貓的小小心願。

**Comment by woohsin** [comment]
我還是喜歡在現實中養貓:)

**Comment by cassy0201** [comment]
感覺能夠滿足現實生活中還不能養貓的期待!

**Comment by 710388** [comment]
太心情了吧，谷歌總是可以推出這麼特別的功能，難怪成為市場第一

**Comment by NcuJerryKao** [comment]
最近Chrome上推出了一個名為「MEOWSING」的網頁版養貓擴充功能，短短一週內就突破了2萬次下載，迅速引起了轟動！這款擴充功能不僅讓你能在上網時與虛擬貓咪互動，還提供了多種功能選項：

不同種類的貓咪：用戶可以選擇自己喜歡的貓咪種類，享受更多樣化的虛擬寵物體驗。
調整貓咪叫聲：這個功能允許用戶根據自己的喜好調整貓咪的叫聲，使體驗更加個人化。
低效能的輕量模式：為了不影響瀏覽器的效能，擴充功能提供了輕量模式，讓你的電腦在有貓咪陪伴的同時，依然能保持高效運行。
這款擴充功能不僅適合愛貓人士，更是上班族的最佳選擇，可以在繁忙的工作中帶來一些趣味和放鬆。你下載了嗎？快來加入這波養貓潮流吧！

**Reply by yubo0oo** [reply]
太誇張了吧烙葛

**Comment by yubo0oo** [comment]
太可愛了
我直接去下載
